### PR TITLE
Organization mute config

### PR DESCRIPTION
**Pull request by thokalavinod**
Related to http://b/330538229.
Issue No: https://b.corp.google.com/issues/339718371

Adding new template to move from SCC API v1 → SCC Management API for post-V2-GA release customers and to support
https://cloud.google.com/security-command-center/docs/reference/rest/v2/organizations.locations.muteConfigs

Release Note Template for Downstream PRs (will be copied)

```release-note:new-resource
`google_scc_organization_mute_config`
```

### CODE REVIEW

**Review by rileykarson — COMMENTED** [review]
Marking review for the timer robot- CI is being used to generate this atm.

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"random_suffix": acctest.RandString(t, 10),
+		"location":      "global",
+	}
+
+	acctest.VcrTest(t, resource.TestCase{
+		PreCheck: func() { acctest.AccTestPreCheck(t) },
+		ProtoV5ProviderFactories: map[string]func() (*schema.Provider, error){
```
Comment (rileykarson) [inline mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go:27]: What are you using as a reference for this line? Please see other tests, which use a predefined factory: https://github.com/hashicorp/terraform-provider-google-beta/blob/bfb8b9067be9965baf389e312815aaa34056fba8/google-beta/services/compute/resource_compute_address_test.go#L35

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go [hunk 2]
```diff
@@ -0,0 +1,124 @@
+package securitycenterv2_test
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google"
```
Comment (rileykarson) [inline mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go:10]: There seems to be an issue importing this. I don't think we generally do import it. See other comments.

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go [hunk 3]
```diff
@@ -0,0 +1,124 @@
...
+  parent         = "organizations/%{org_id}"
+  filter         = "category: \"OS_VULNERABILITY\""
+  location       = "%{location}"
+  description    = "My Mute Config"
+  type           = "STATIC"
+}
+`, context)
+}
+
+func testAccCheckSecurityCenterOrganizationMuteConfigDestroyProducer(t *testing.T) resource.TestCheckFunc {
```
Comment (rileykarson) [inline mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go:97]: The generated test file will contain a DestroyProducer. Check google-beta/services/securitycenterv2/resource_scc_v2_organization_mute_config_generated_test.go i.e. https://github.com/modular-magician/terraform-provider-google-beta/compare/auto-pr-10989-old..auto-pr-10989#diff-5b7cf01d02c77b0af605b1c63bd222a8e7283b56cdf289e6fbbbb878a21e07ef

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go (hunk 1 above)
Comment (thokalavinod) [reply mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go]: addressed

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go (hunk 2 above)
Comment (thokalavinod) [reply mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go]: addressed

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go (hunk 3 above)
Comment (thokalavinod) [reply mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go]: addressed

File: mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google-beta/google-beta"
+    "github.com/hashicorp/terraform-provider-google-beta/google-beta/acctest"
+    "github.com/hashicorp/terraform-provider-google-beta/google-beta/envvar"
```
Comment (rileykarson) [inline mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go:12]:
```suggestion
	"github.com/hashicorp/terraform-provider-google/google"
	"github.com/hashicorp/terraform-provider-google/google/acctest"
	"github.com/hashicorp/terraform-provider-google/google/envvar"
```

Comment (thokalavinod) [reply mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_organization_mute_config_test.go]: addressed

**Comment by rileykarson** [comment]
Obsoleted by https://github.com/GoogleCloudPlatform/magic-modules/pull/11168

**Pull request closed without merging**